index.html

```diff
@@ -0,0 +1,12 @@
+<html>
+<head><title>Athen</title><style>
+  ::placeholder {
+  size=5px;
+  opacity: 1;
+  </style></head>
+<center><body bgcolor=#003366>
+  <button>login|register</button>
+<br><br><br><br><br><br><br><br><br><br><br><br>

```

Comment:
violation of the prettier style guidelines, as all the <br> tags cannot be in the same line

Reply:
@arjunkapoor19 Please fix the style violations!

Reply:
right away

Reply:
done @sayamkanwar 

p.s. i have not received any marks for other submissions

Reply:
Which submission?